sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample2_RecognizeCustomForms.md

```diff
@@ -1,6 +1,6 @@
 # Recognize custom forms
 
-This sample demonstrates how to recognize form fields and other content from your custom forms, using models you trained with your own form types. For more information on how to do the training, see [train a model][train_a_model].
+This sample demonstrates how to recognize form fields and other content from your custom forms, using models you trained with your own form types. For more information on how to do the training, see [train a model][train_a_model]. For a suggested approach to extracting information from recognized custom forms with known fields, see [strongly-typing a recognized form][strongly_typing_a_recognized_form].
```

Comment:
>For a suggested approach to extracting information from custom forms with known fields, see [strongly-typing a recognized form][strongly_typing_a_recognized_form].

Not sure if it is clear o the user that `strongly typing a recognized form` is only available for a specific set of fields or even receipt fields for that matter. Should we consider making that more clear?

Reply:
The sample is intended to be an approach that can be used for any custom form: create a wrapper class, make your fields strong-typed and present information in a way that's easily accessible. We're using receipts as an example because that's the only pre-trained model we can use, but we make it clear in the sample that the same approach can be followed for other forms.

I have updated the sample a lot since your comment, so I may already have addressed your concerns.